src/Analysim.Web/ClientApp/src/app/email-confirmation/email-confirmation.component.ts

```diff
@@ -17,9 +18,12 @@ export class EmailConfirmationComponent implements OnInit {
     private route: ActivatedRoute,
     private accountService: AccountService,
     private modalService: BsModalService,
+    public notfi: NotificationService
   ) { }
 
   async ngOnInit(): Promise<void> {
+    this.router.navigate(['/login']);
+    this.notfi.showInfo('Email Confirmation Successful','You can now login!');
```

Comment:
![image](https://user-images.githubusercontent.com/78971883/192671874-32a33450-4537-4f20-b38c-801af0c3f9d1.png)
The api only links to that page if it is successful, otherwise it goes to the error page.

Reply:
yeah, we need to fix this later.